src/modules/commander/commander.cpp

```diff
@@ -2498,15 +2506,34 @@ int commander_thread_main(int argc, char *argv[])
 			main_state_before_rtl == commander_state_s::MAIN_STATE_RATTITUDE ||
 			main_state_before_rtl == commander_state_s::MAIN_STATE_STAB)) {
 
-			// transition to previous state if sticks are increased
-			const float min_stick_change = 0.2f;
+			// transition to previous state if sticks are touched
 			if ((_last_sp_man.timestamp != sp_man.timestamp) &&
 				((fabsf(sp_man.x) - fabsf(_last_sp_man.x) > min_stick_change) ||
```

Comment:
If I read it correctly this is handled like a switch transition, meaning the value is compared to what it was when the fc last received one. So the actual speed with which you have to move your stick depends on the RC update rate and if you happen to have a high update frequency you might never reach such a step.